infra/base-images/base-builder/Dockerfile

```diff
@@ -43,8 +43,8 @@ RUN apt-get install -y \
         libreadline-dev \
         libffi-dev \
         libbz2-dev \
-        liblzma-dev
-RUN cd /tmp/ && \
+        liblzma-dev && \
```

Comment:
should we also remove the rest of these packages?

Reply:
Done!